gateway/ln-gateway/src/lightning/lnd.rs

```diff
@@ -1134,7 +1134,7 @@ impl ILnRpcClient for GatewayLndClient {
         let mut client = self.connect().await?;
 
         // Connect to the peer first
-        client
+        let _ = client
```

Comment:
Why is this being swallowed?

Reply:
I'm still trying to figure out why CI is failing, this was one of the things I tweaked that I thought might fix it. The intention here was to make this function idempotent, since we retry channel opens in devimint:

https://github.com/fedimint/fedimint/blob/004c35d4b1caaa33c17f7640d5f17f93c3547154/devimint/src/external.rs#L915-L919

I believe calling `ConnectPeer` on a peer that's already connected returns an error.